tests/etl/test_buildbot.py

```diff
@@ -1274,6 +1285,17 @@
                    'os': 'mac',
                    'os_platform': 'osx-10-6',
                    'vm': False}}),
+    ('Rev4 MacOSX Snow Leopard 10.6 mozilla-central talos g2-snow-e10s',
+     {'build_type': 'opt',
+      'job_type': 'talos',
+      'name': {'group_name': 'Talos Performance e10s',
+               'group_symbol': 'T-e10s',
+               'name': 'Talos g2 e10s',
+               'job_symbol': 'g2'},
+      'platform': {'arch': 'x86_64',
+                   'os': 'mac',
+                   'os_platform': 'osx-10-6',
+                   'vm': False}}),
```

Comment:
note: we turned off all 10.6 talos tests- we won't be scheduling this on 10.6, so this clause might be optional.